app/controllers/api/students_controller.rb

```diff
@@ -22,6 +24,23 @@ def show
     render json: student
   end
 
+  def unmatched
+    matchings = Matching.all
+    student_ids = Set.new
+    for matching in matchings do
+      student_ids.add matching.student_id
+    end
+
```

Comment:
Dont forget each student can have multiple instruments! so you would have to check which of the students' instruments aren't in a matching object.

I'm changing the instrument attributes into strings right now so just code it as if they were strings